backend/djangoindia/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for djangoindia project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'djangoindia.settings')
```

Comment:
```suggestion
os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'djangoindia.settings.local')
```

Reply:
We don't really need to change this one, instead it should be `wsgi.py`